package.json

```diff
@@ -48,7 +48,8 @@
   },
   "peerDependencies": {
     "@babel/core": "7.x",
-    "eslint": ">=7.27.x"
+    "eslint": ">=7.27.x",
+    "typescript": ">=4"
```

Comment:
needed to add `typescript`, otherwise the rule wasn't working.
Raising this error: `error  Definition for rule '@typescript-eslint/no-explicit-any' was not found  @typescript-eslint/no-explicit-any`

Reply:
yeah, I guess it does, I'll update the PR description